torchsde/_core/misc.py

```diff
@@ -152,9 +152,23 @@ def batch_mvp(m, v):
     return mvp
 
 
-def grad(inputs, **kwargs):
+def grad(outputs, inputs, grad_outputs=None, **kwargs):
     # Workaround for PyTorch bug #39784
+    outputs = make_seq_requires_grad(outputs)
```

Comment:
I don't think we should pack additional functionality into `grad`. I think of it as just `autograd.grad` + workaround for bugfix, and it should be up to the caller whether they want  to do this.

Reply:
I think this pack is reasonable, as I found it to reduce a lot of repeated code. The function will just be used internally, and at all the places where I've looked at, it seems to make sense.

Reply:
Hmm, okay. With luck this might be one of the things that comes out in the tuple-to-tensor rewrite anyway.